packages/flutter/test/material/search_anchor_test.dart

```diff
@@ -1741,9 +1794,79 @@ void main() {
     await tester.tap(find.widgetWithIcon(IconButton, Icons.search));
     await tester.pumpAndSettle();
 
-    final SizedBox sizedBox = tester.widget<SizedBox>(find.descendant(of: findViewContent(), matching: find.byType(SizedBox)).first);
-    expect(sizedBox.width, 280.0);
-    expect(sizedBox.height, 390.0);
+    final Size size = tester.getSize(find.descendant(of: findViewContent(), matching: find.byType(ConstrainedBox)).first);
+    expect(size.width, 280.0);
+    expect(size.height, 390.0);
+  });
+
+  testWidgets('SearchAnchor respects viewBarPadding property', (WidgetTester tester) async {
```

Comment:
I think this test is duplicated. Maybe this happened in some reverts?

Reply:
This comment seems not resolved? Can we remove this unit test since it already exists below.